### PR TITLE
TC yaml update

### PR DESCRIPTION
**Pull request by thiagoftsm**
##### Summary
Add description for `tc`  plugin.

##### Test Plan

1. Take a look in CI and grammar.

##### Additional Information

<details> <summary>For users: How does this change affect me?</summary>
Describe the PR affects users: 
- Which area of Netdata is affected by the change? documentation, integration and dashboard.
- Can they see the change or is it an under the hood? If they can see it, where? no
- How is the user impacted by the change? a plugin well documented.
- What are there any benefits of the change? A more clear information for our users.
</details>

### CODE REVIEW

File: collectors/tc.plugin/metadata.yaml [hunk 1]
```diff
@@ -1,53 +1,61 @@
...
         enabled: true
-      list: []
+      list:
+        - name: script to run to get tc values
+          description: Path to script `tc-qos-helper.sh`
+          default_value: "usr/libexec/netdata/plugins.d/tc-qos-helper.s"
+          required: false
     examples:
       folding:
         enabled: true
```
Comment (sashwathn) [inline collectors/tc.plugin/metadata.yaml:57]: No examples for configuration?

**Review by sashwathn — APPROVED** [review]

**Comment by sashwathn** [comment]
@thiagoftsm : What was the need to convert all ' to "? Is this to take care of some yaml error?
At least all our `metadata.yaml`s still use '.

**Comment by Ancairon** [comment]
>@thiagoftsm : What was the need to convert all ' to "? Is this to take care of some yaml error?
At least all our metadata.yamls still use '.

@sashwathn it is common to format the file before editing, so indentation is correct, vscode changes these by default, either work.

**Review by Ancairon — COMMENTED** [review]
It is not exactly clear if the "additional permissions" are satisfied by giving the path in the configuration, or if there is any other prerequisite. This is a place where we might want to keep the README if eventually we produce markdown files, as it has a lot of information.

**Review by sashwathn — APPROVED** [review]

**Review by shyamvalsan — APPROVED** [review]

**Pull request merged**